app/api/__init__.py

```diff
@@ -26,11 +27,12 @@
 api.route(EventList, 'event_list', '/events')
 api.route(EventDetail, 'event_detail', '/events/<int:id>', '/tickets/<int:ticket_id>/event',
           '/microlocations/<int:microlocation_id>/event', '/social_links/<int:social_link_id>/event',
-          '/sponsors/<int:sponsor_id>/event')
+          '/sponsors/<int:sponsor_id>/event', '/tracks/<int:id>/event')
```

Comment:
No. I've used id in https://github.com/fossasia/open-event-orga-server/pull/3713/files#diff-b38e5fc3b36c615183eeb17dd03f0317R32 so this should be id.